.github/workflows/_bitrise-check.yml

```
@@ -0,0 +1,53 @@
+name: "Bitrise Status Check"
```

Head commit status Check
```suggestion
name: "Branch Head Commit Status Check"
```